src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -43,7 +43,7 @@ const useFetchChannelHook = () => {
           : getChatName(channel?.name, signedChannelUsername);
 
       if (channel?.type === 'group' || channel?.type === 'topics') {
-        signedChannelImage = channel?.channel_image;
+        signedChannelImage = channel?.channel_image ?? channel.image;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here introduces a fallback to `channel.image` if `channel.channel_image` is not available. This seems like a good improvement for error handling and ensuring that an image is always displayed when possible.
```diff
- signedChannelImage = channel?.channel_image;
+ signedChannelImage = channel?.channel_image ?? channel.image;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->